spot-ml/src/main/scala/org/apache/spot/SuspiciousConnectsArgumentParser.scala

```diff
@@ -45,6 +46,10 @@ object SuspiciousConnectsArgumentParser {
       action((x, c) => c.copy(topicCount = x.toInt)).
       text("topic count")
 
+    opt[String]("userdomain").required().valueName("<user domain>").
```

Comment:
ok that makes sense to me too at the argument parser level. But should we still keep it as passing through spot.conf so that the decision is located in  a config file (with default value an empty string)? Or tell people to go in and change the scala code?